AutoCalculator/Scenes/Main/CurrencyConverter/CurrencyViewModel.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  CurrencyViewModel.swift
+//  AutoCalculator
+//
+//  Created by 강성찬 on 2023-06-16.
+//
+
+import Foundation
+
+class CurrencyViewModel: ObservableObject {
+
+    @Published var input = ""
+    @Published var base = ""
+    @Published var currencyList = [String]()
+    
+    private var networkManager = NetworkManager()
+
+    func makeRequest(showAll: Bool, currencies: [String] = ["USD", "GBP", "EUR"]) {
+        networkManager.apiRequest(url: "https://api.exchangerate.host/latest?base=\(base)&amount=\(input)") { currency in
```

Comment:
url would be nice to be handled some sort of NetworkManager instead of ViewModel handling it